templates/assets/cloudformation/elb.yml

```diff
@@ -210,21 +219,14 @@ Resources:
       Tags:
       - Key: Name
         Value: !Ref AWS::StackName
+      - !If [ShouldCreateElbAccessLogsS3Bucket, {"Key": 'mu:bucketpolicy', "Value": {"Ref": "ElbAccessLogsS3BucketPolicy"}}, !Ref 'AWS::NoValue']
```

Comment:
Stylistically can you have this conditional match the higher line count format? It is easier to discern what is going on.

Reply:
PR author's comment on this at https://github.com/stelligent/mu/pull/428#discussion_r272642746